rubix/core/pipeline.py

```diff
@@ -79,30 +82,27 @@ def _prepare_data(self) -> dict:
         """
         # Get the data
         self.logger.info("Getting rubix data...")
-        coords, velocities, metallicity, mass, age, halfmassrad_stars = get_rubix_data(
-            self.user_config
+        rubixdata = get_rubix_data(self.user_config)
+        star_count = (
+            len(rubixdata.stars.coords) if rubixdata.stars.coords is not None else 0
+        )
+        gas_count = len(rubixdata.gas.coords) if rubixdata.gas.coords is not None else 0
+        self.logger.info(
+            f"Data loaded with {star_count} star particles and {gas_count} gas particles."
         )
-        self.logger.info(f"Data loaded with {len(coords)} particles.")
+        self.logger.info(f"Data loaded with {sys.getsizeof(rubixdata)} properties.")
```

Comment:
maybe we can implement a function insidee the rubix data object that returns the number of properties.

Reply:
Sure, feel free to implement it

Reply:
let me suggest to reference this in the [equinox module issue](https://github.com/ufuk-cakir/rubix/issues/119)
and we think about it when we restructure the RubixData class anyways.